roles/dotnet/handlers/verify.yml

```diff
@@ -0,0 +1,11 @@
+- name: Wait for AppDynamics.Agent.Coordiantor to start
+  win_wait_for_process:
+    process_name_pattern: 'AppDynamics.Coordinator'
+    state: present
+    timeout: 60
+
+- name: Tail .NET agent logs
+  win_shell: 'powershell "Get-Content {{ DOTNETAGENTFOLDER }}\Logs\AgentLog.txt -Tail 30"'
```

Comment:
`powershell` is going away and it's not available in none server core windows servers anymore.. it's being replaced with `pwsh` which is the equivalent fo PowerShell core.  I'd suggest you remove `powershell` term. Have you tried `win_command`?

Reply:
Thanks, fixed!